src/StimulusBundle/src/Dto/StimulusAttributes.php

```diff
@@ -212,18 +212,22 @@ private function getFormattedValue(mixed $value): string
         return (string) $value;
     }
 
-    private function escapeAsHtmlAttr(mixed $value): string
+    private function escape(mixed $value, string $strategy): string
     {
-        if (class_exists(EscaperRuntime::class)) {
-            return $this->env->getRuntime(EscaperRuntime::class)->escape($value, 'html_attr');
-        }
-
-        if (method_exists(EscaperExtension::class, 'escape')) {
-            return EscaperExtension::escape($this->env, $value, 'html_attr');
+        static $escaper;
+
+        if (!$escaper) {
```

Comment:
Quick win... I think? I don't wanted to re-execute `N` times the escaping logic, when calling `N` times `$this->escape()`.

But I believe `class_exists` is fast enough to be re-executed multiple times. 
I'm gonna revert